---
float64 heading
float64 pitch
float64 roll
float64 latitude
float64 longitude
float64 altitude